### PR TITLE
Improved descriptions and links

### LINKED ISSUE #109: Bad looking cross references
On this page: https://www.cyberbotics.com/doc/reference/robot#wb_robot_get_device
The **See also** &rarr; **section** looks bad. Instead of **section**, we should display **wb_robot_step**.

### PR DESCRIPTION
**Pull request by omichel**
Fixed #109 and improved some descriptions.

### CODE REVIEW

**Comment by omichel** [comment]
@fabienrohrer: do you agree we should manually change all the "See also" sections?

**Comment by omichel** [comment]
Well, it turns out such sections were only wrong in robot.md and nowhere else.

**Comment by fabienrohrer** [comment]
Yes, they seems wrong, and only a manual modification can solve this.

File: reference/robot.md [hunk 1]
```diff
@@ -356,9 +359,12 @@ function `getDistanceSensor` will return a reference to a
 found, the function returns `NULL` in C++, `null` in Java or the `none` in
 Python.
 
+> Note: These functions are not available in the C and MATLAB APIs. Instead, C and Matlab users should use [wb\_robot\_get\_device](#wb_robot_get_device).
```
Comment (omichel) [inline reference/robot.md]: The rendering of this link is wrong: see https://www.cyberbotics.com/doc/reference/robot?version=fix-robot-doc#wb_robot_get_device

robot appears in *italics* and some underscores are missing...

Comment (omichel) [inline reference/robot.md]: @fabienrohrer: do you know what is the problem?

Comment (omichel) [inline reference/robot.md]: Sorry. It seems to work now. It was probably caused by a cache problem.

**Comment by omichel** [comment]
:+1:

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**